assets/_sass/custom-viz/boeing/_scrollytelling.scss

```diff
@@ -0,0 +1,29 @@
+/*==============================
+=        Scrollytelling        =
+==============================*/
+
+.label-group {
+  transition: opacity 1s linear;
+}
+
+.hidden {
+  opacity: 0;
+  transition: opacity 1s linear;
+}
+
+.display-none {
+  display: none;
+  transition: opacity 1s linear;
+}
+
+.boeing-js__opacity-not-viewed {
+  opacity: .5;
```

Comment:
Numbers less than 1 should have leading zeros --> `0.5` re our coding style guide. :) I know we don't have it set up on Trade Guys, but let's try and stay consistent with it across projects.